app/models/ems_refresh/save_inventory.rb

```diff
@@ -1,8 +1,12 @@
 module EmsRefresh::SaveInventory
   def save_ems_inventory(ems, hashes, target = nil)
+    if hashes.kind_of?(Array)
+      ManagerRefresh::SaveInventory.save_inventory(ems, hashes)
+      return
+    end
     if hashes && hashes[:_inventory_collection]
```

Comment:
right, then we can delete this one, once I will rewrite it in AWS and @agrare in VmWare, passing :_inventory_collection was hacky anyway :-)